static/js/tools/choropleth/choropleth.tsx

```diff
@@ -0,0 +1,356 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Creates and manages choropleth rendering.
+ */
+
+import React, { Component } from "react";
+import axios from "axios";
+import * as d3 from "d3";
+
+class ChoroplethMap extends Component {
+  constructor() {
+    super({});
+
+    // Redirect to basic url if none provided.
+    if (window.location.search === "") {
+      window.location.search =
+        "statVar=Count_Person_Employed&pc=t&geoDcid=country/USA";
+    }
+
+    // Add default state and bind function contexts.
+    const urlParams = new URLSearchParams(window.location.search);
+    this.state = {
+      geojson: [],
+      pc:
+        urlParams.has("pc") &&
```

Comment:
What I meant here was:
if urlParams.has("pc") ==> pc is true
if the param isn't present, then treat it as false. Then there's no need to compare to one of 3 values

Reply:
I think that this would complicate the link generation on the front-end.

Reply:
I will add a TODO to look into it. That could be nice.